opslevel/terraform_type_conversions.go

```diff
@@ -47,3 +47,8 @@ func unquote(value string) string {
 	}
 	return value
 }
+
+// Returns value wrapped in a types.Int64Value
+func RequiredInt64Value(value int) basetypes.Int64Value {
```

Comment:
Is this used?

Reply:
Not yet. In the end this is syntactic sugar and since it's not clear what we would sensibly pass into an `OptionalInt64Value` I think I'm going to remove this